Added bootstrap elements to remaining pages, fixed nav/image links.

DyPickard opened the pull request:
This PR is for issues #82 and #78 

- Added more bootstrap styling to about, admin, and search page.
- created grid system for search page to more easily order elements.
- Fixed nav links and image links.
- Created event info table for search page

anthonyargatoff reviewed (changes requested):
Is it possible to force the footer to the bottom of the page? On my browser it is floating near the middle of the screen (or is this intentional)?
![image](https://github.com/anthonyargatoff/LifeCycle-Thugs/assets/122230360/b7c91574-13cb-4725-a126-8ff92dd26413)

On the search bar, can you remove the horizontal scroll bar?
![image](https://github.com/anthonyargatoff/LifeCycle-Thugs/assets/122230360/c8bbe6cd-caa6-472c-b8a2-868131156a45)

All of the other changes look good to me.

DyPickard commented:
Oh, I didn't notice that the footer was floating. Yeah I can fix that.

The scroll bar was intentional. If we search a location that has a large number of events, the page will get longer because it will list all items on the page with no overflow. I could try to figure out if I can hide the scroll bar unless overflow is needed. If you guys think it should be different somehow though I can change it!

anthonyargatoff commented:
The vertical scroll bar is fine, but the greyed out horizontal bar at the bottom seems unnecessary right?

DyPickard commented:
Oh i see what you mean, yes ill change that

DyPickard commented:
Fixed horizonal scroll
<img width="353" alt="image" src="https://github.com/anthonyargatoff/LifeCycle-Thugs/assets/110944869/9d121e1b-21d5-4288-97c5-c16e55f8bb33">

footer was high up just because we don't have much content on each page. I added a 100% viewport container to each page to fix the issue, so footer will always be below the fold.
<img width="1033" alt="image" src="https://github.com/anthonyargatoff/LifeCycle-Thugs/assets/110944869/8ea9ba76-aad8-4571-ba7d-c89f09b0872f">


also added active menu items to indicate which page on
<img width="299" alt="image" src="https://github.com/anthonyargatoff/LifeCycle-Thugs/assets/110944869/78d8c170-8e81-4c46-a1cd-92d6bcf64909">

anthonyargatoff reviewed (approved):
Excellent. Changes look rad

DyPickard commented:
I added a reference image for search page layout since I don't have the map element yet.

RyanPybus reviewed (approved):
Looks very good, changes are good
